Gruntfile.js

```diff
@@ -575,6 +584,29 @@ module.exports = function (grunt) {
         ]);
     });
 
+    grunt.registerMultiTask('swPrecache', function() {
+        var done = this.async();
+        var rootDir = this.data.rootDir;
+        var files =  (function() {
+            return (require('./sw-cache-file-list.json')).files;
+        }());
+
+        var config = {
+            cacheId: 'bramble-cache::' + Date.now(),
+            logger: grunt.log.writeln,
+            staticFileGlobs: files,
+            stripPrefix: 'dist/',
+            ignoreUrlParametersMatching: [/./]
+        };
+
+        swPrecache.write(Path.join(rootDir, 'bramble-sw.js'), config, function(err) {
```

Comment:
That's what it's doing, see https://github.com/mozilla/brackets/pull/547/files/46deab8a7cddc92bbcb6683675ec4e2c5aa65b1b#diff-35b4a816e0441e6a375cd925af50752cR528.